src/constants.js

```diff
@@ -1,4 +1,5 @@
 export const CLAIM_WITS_PER_ADDRESS = 1
+export const CLAIMING_MIN_WITS_ADDRESS = 50
```

Comment:
This name seems to imply that this is an address, because the unit is usually the last word. Maybe rename to:
```suggestion
export const CLAIMING_ADDRESS_MIN_WITS = 50
```